Expand notes, change formatting, and update priorities

chloecinders opened the pull request:
- Expands the notes to avoid any confusion
- Moves info from priorities to notes
- Adds "Large bots" priority as they are different from verified and unverified bots
- Changes formatting to be more consistent
- Rewords some sentences

chloecinders committed:
Expand notes, change formatting, and update priorities

Tolga1452 commented on experiment-rollout.js:
```diff
@@ -46,33 +46,45 @@ const data = {
             },
             {
                 status: 2,
-                name: 'Discord Staff'
+                name: 'Large bots | Criteria unknown | Reservations only'
             },
             {
                 status: 2,
-                name: 'Partnered, subscription-enabled (100USD/mo+), and verified server owners (includes some old partners + some content creators)'
+                name: 'Discord Staff | Staff and personal accounts'
             },
```
wdym by "large bots" if they're not verified? also why the order has changed?

chloecinders replied:
There was another priority for large bots, i.e. Dyno and Mee6 who got their names reserved but not yet changed. This was also directly after brands and that's why the order changed.

Tolga1452 replied:
ah okay got it

Tolga1452 commented on experiment-rollout.js:
```diff
@@ -46,33 +46,45 @@ const data = {
-                status: 0,
-                name: 'Top (likely 100, by server count) bot developers (Unconfirmed)'
+                status: 1,
+                name: 'Verified bots | Ordered by support server member count'
```
where did you get that "support server member count" info?

chloecinders replied:
Thats from the "Server count wins name conflicts, other ones will get their discriminator appended", reworded

Tolga1452 replied:
but "server count", not "member count in support server"

chloecinders replied:
I misinterpreted the original priority name - fixed

Tolga1452 commented on experiment-rollout.js:
```diff
@@ -46,33 +46,45 @@ const data = {
+            },
+            {
+                title: 'Pomelo check bots',
+                text: 'Pomelo check bots operate on either the Friend request api or on lists/databases made before the API endpoint gate. This means that they are inaccurate.'
```
```suggestion
                text: 'Pomelo check bots operate on either the Friend request API or on lists/databases made before the API endpoint gate. This means that they are inaccurate.'
```

Tolga1452 commented on experiment-rollout.js:
```diff
@@ -46,33 +46,45 @@ const data = {
+            },
+            {
+                title: 'Bot name conflicts',
+                text: 'if two or more bots have the same name the bot with the least members in its support server will get the name. The other ones will get their discriminator appended to their name. I.e. @bot1234'
```
```suggestion
                text: 'If two or more bots have the same name the bot with the least members in its support server will get the name. The other ones will get their discriminator appended to their name. I.e. @bot1234'
```

also "least members in its support server" again?

chloecinders replied:
fixed

chloecinders committed:
Add back server subscriptions priority and fix typos

evnsh commented:
Hello!

Could you elaborate on the following:
> Brands | Reserved unless Discord partner

Discord Partner aren't able to choose the username that they want, they have a specific username that is reserved to them, generally a variation of their current username and aren't able to claim brand names which are blacklisted by Discord.

> Large bots | Criteria unknown | Reservations only

Large bots would imo fall into the Brand category, and so far no "large bots" have been able to claim their username (most of them are however reserved and cannot be picked by another user)

> Verified bots | Ordered by support server member count

Where does the support server member count come from? Again large bots like MEE6 or YAGPDB have not been given access to Pomelo yet.

> Everyone else | Ordered by Nitro/Account creation

Why did you remove the March 1st date for Nitro? This seems pretty important imo.

chloecinders committed:
Fix capitalization

Co-authored-by: ✨Tolgchu✨ <tolgchu-studios@outlook.com>

chloecinders committed:
Update experiment-rollout.js

Tolga1452 commented on experiment-rollout.js:
```diff
@@ -46,33 +46,53 @@ const data = {
             {
                 status: 1,
-                name: 'Verified bots (Server count wins name conflicts, other ones will get their discriminator appended)'
+                name: 'Verified bots | Ordered by guild count'
```
```suggestion
                name: 'Verified bots | Ordered by server count'
```

"server" can be clearer

chloecinders commented:
> Hello!

Hey!
 
> Could you elaborate on the following:
> 
> > Brands | Reserved unless Discord partner
>
> Discord Partner aren't able to choose the username that they want, they have a specific username that is reserved to them, generally a variation of their current username and aren't able to claim brand names which are blacklisted by Discord.

Yep, that's why I added "Reserved unless Discord partner". Brands like Youtube and TikTok have their names reserved, not claimed. Other brands also had their names reserved but due to having a Partnered brand account (i.e. the brand account is a server owner) they could claim their name.

> > Large bots | Criteria unknown | Reservations only
> 
> Large bots would imo fall into the Brand category, and so far no "large bots" have been able to claim their username (most of them are however reserved and cannot be picked by another user)

Agreed. Included it in the brand category. Also the "Reservations only" again, bot developers have come out and said that they can reserve a name they want. They cant claim it though. That's why I added the Reservations only.

> > Verified bots | Ordered by support server member count
> 
> Where does the support server member count come from? Again large bots like MEE6 or YAGPDB have not been given access to Pomelo yet.

Changed to Ordered by guild count.

> > Everyone else | Ordered by Nitro/Account creation
> 
> Why did you remove the March 1st date for Nitro? This seems pretty important imo.

Added it back into notes.

Tolga1452 commented on experiment-rollout.js:
```diff
@@ -46,33 +46,53 @@ const data = {
-                name: 'Verified bots (Server count wins name conflicts, other ones will get their discriminator appended)'
+                name: 'Verified bots | Ordered by guild count'
             },
             {
                 status: 0,
-                name: 'Top (likely 100, by server count) bot developers (Unconfirmed)'
-            },
```
where did this go?

chloecinders replied:
Its unconfirmed, so I thought it was unnecessary? People take that stuff as face value and spread it as if it were 100% confirmed by Discord staff.

Tolga1452 replied:
yeah keeping it untill 100% confirmation will be better

chloecinders committed:
Change guild to server

Co-authored-by: ✨Tolgchu✨ <tolgchu-studios@outlook.com>

Tolga1452 reviewed (approved):
i think it's ready to merge now?

evnsh commented:
> Partnered / verified / ex-partnered / server subscription server owners

Not every ex-partner gets Pomelo, this needs to be clearer.

> Everyone else | Ordered by Nitro/Account creation

Please put back the 1st March date for Nitro it's important.

Rollout to Verified Bots hasn't started afaik, the status should be 0.

Tolga1452 commented:
> Please put back the 1st March date for Nitro it's important.

done ✅

chloecinders committed:
Update rollout status for verified bots

Tolga1452 commented:
> Rollout to Verified Bots hasn't started afaik, the status should be 0.

you set it in the pr lol. but i can confirm it hasn't started yet

chloecinders commented:
Changed status to 0

chloecinders committed:
Change ex-partners and MPA to note

chloecinders committed:
Fix capitalization
